### ISSUE TITLE
Let's get KDE plasma mobile running!

### ISSUE THREAD

**Issue by ollieparanoid** [postmarketOS/pmbootstrap#378]
Follow up to *Package Plasma Mobile* #161.

The current status is, that `kwin` does not run yet, and we're working together with KDE's developers in this ticket: https://bugs.kde.org/show_bug.cgi?id=383486

Additionally, the dependency `dbus-launch` hangs for a minute or so, see #377.

**Comment by PureTryOut** [comment]
We got `kwin` (and `postmarketos-demos` on it) to run succesfully! 🎉 

![screenshot_2017-08-19_14-56-20](https://user-images.githubusercontent.com/1565836/29486793-aa85d30e-84ee-11e7-92be-de6e26ffc1d5.png)

Note that the mouse is stuck, as for some reason kwin_libinput doesn't detect the mouse, as a mouse. Quite strange, I know.

But a lot of progress! Next up is getting the mouse working, and then launching the Plasma Shell. Should be *relatively* smooth sailing from here on out.

**Comment by PureTryOut** [comment]
It now runs!

[Plasma shell running on postmarketOS](https://zippy.gfycat.com/LikablePoliteHog.webm)

As you can see in the video, there is still some work to do, but we're getting really close!

**Comment by zhuowei** [comment]
@PureTryOut I tested your branch on my Nexus 6P (checked out postmarketos-ui-plasma-mobile branch, rebased on top of master, fixed a few checksum issues and kinit's recursive dependency (?)). Found two issues:

- ConsoleKit isn't started at boot, so kwin hangs forever.
- After I manually start console-kit-daemon, kwin exits with
"FATAL ERROR: backend failed to initialize, exiting now"

The last line before this in `strace` was:
`ioctl(21, FBIOGET_VSCREENINFO, 0x7fe2332c80) = 0`

so it looks like it hates my framebuffer.
